src/data/inchi-dictionary.ts

```diff
@@ -2,430 +2,825 @@ import { INCHIProps } from '../interfaces/utils';
+    CAS: '',
     inchiString: '',
     inchiKey: '',
     molecularWeight: 272.21,
   },
+
   AmmoniumChlorideSalammoniac: {
+    CAS: '12125-02-9',
     inchiString: 'InChI=1S/ClH.H3N/h1H;1H3',
     inchiKey: 'NLXLAEXVIDQMFP-UHFFFAOYSA-N',
     molecularWeight: 53.49,
   },
+
   AmmoniumChloroplatinate: {
+    CAS: '16919-58-7',
     inchiString: 'InChI=1S/6ClH.2H3N.Pt/h6*1H;2*1H3;/q;;;;;;;;+4/p-4',
     inchiKey: 'PCCGQTHFYHJATL-UHFFFAOYSA-J',
     molecularWeight: 443.87,
   },
+
   AmmoniumChloroplatinite: {
+    CAS: '13820-41-2',
     inchiString: 'InChI=1S/4ClH.2H3N.Pt/h4*1H;2*1H3;/q;;;;;;+2/p-2',
     inchiKey: 'QJIMNDWDOXTTBR-UHFFFAOYSA-L',
-    molecularWeight: 372.97, // 373.0 # https://pubchem.ncbi.nlm.nih.gov/compound/16211508
+    molecularWeight: 372.97,
   },
+
   AmmoniumChlorostannate: {
+    CAS: '',
     inchiString: 'InChI=1S/6ClH.2H3N.Sn/h6*1H;2*1H3;/q;;;;;;;;+4/p-4',
     inchiKey: 'UDMAVAHFQCVSSE-UHFFFAOYSA-J',
     molecularWeight: 367.5,
   },
+
   AmmoniumChromate: {
+    CAS: '',
     inchiString: 'InChI=1S/Cr.2H3N.4O/h;2*1H3;;;;/q;;;;;2*-1/p+2',
     inchiKey: 'MFFLHUNPSHBKRG-UHFFFAOYSA-P',
     molecularWeight: 152.07,
   },
+
   AmmoniumCyanide: {
+    CAS: '12211-52-8',
     inchiString: 'InChI=1S/CN.H3N/c1-2;/h;1H3/q-1;/p+1',
     inchiKey: 'ICAIHGOJRDCMHE-UHFFFAOYSA-O',
     molecularWeight: 44.06,
   },
+
   AmmoniumDichromate: {
+    CAS: '7789-09-5',
     inchiString: 'InChI=1S/2Cr.2H3N.7O/h;;2*1H3;;;;;;;/q;;;;;;;;;2*-1/p+2',
     inchiKey: 'JOSWYUNQBRPBDN-UHFFFAOYSA-P',
     molecularWeight: 252.06,
   },
+
   AmmoniumFerrocyanide: {
+    CAS: '',
     inchiString: '',
     inchiKey: '',
     molecularWeight: 392.19,
   },
+
   AmmoniumFluoride: {
+    CAS: '12125-01-8',
     inchiString: 'InChI=1S/FH.H3N/h1H;1H3',
     inchiKey: 'LDDQLRUQCUTJBB-UHFFFAOYSA-N',
     molecularWeight: 37.04,
   },
+
   AmmoniumFluorideAcid: {
+    CAS: '1341-49-7',
     inchiString: 'InChI=1S/2FH.H3N/h2*1H;1H3',
     inchiKey: 'KVBCYCWRDBDGBG-UHFFFAOYSA-N',
     molecularWeight: 57.04,
   },
+
   AmmoniumFormate: {
+    CAS: '540-69-2',
     inchiString: 'InChI=1S/CH2O2.H3N/c2-1-3;/h1H,(H,2,3);1H3',
     inchiKey: 'VZTDIZULWFCMLS-UHFFFAOYSA-N',
     molecularWeight: 63.06,
   },
+
   AmmoniumHydrosulfide: {
+    CAS: '',
     inchiString: 'InChI=1S/H3N.H2S/h1H3;1H2',
     inchiKey: 'HIVLDXAAFGCOFU-UHFFFAOYSA-N',
     molecularWeight: 51.11,
   },
+
   AmmoniumHydroxide: {
+    CAS: '12168-30-8',
     inchiString: 'InChI=1S/H3N.H2O/h1H3;1H2',
     inchiKey: 'VHUUQVKOLVNVRT-UHFFFAOYSA-N',
     molecularWeight: 35.05,
   },
+
   AmmoniumMolybdate: {
+    CAS: '13106-76-8',
     inchiString: 'InChI=1S/Mo.2H3N.4O/h;2*1H3;;;;/q;;;;;2*-1/p+2',
     inchiKey: 'APUPEJJSWDHEBO-UHFFFAOYSA-P',
     molecularWeight: 196.01,
   },
+
   AmmoniumHeptamolybdate: {
+    CAS: '',
     inchiString:
       'InChI=1S/7Mo.6H3N.10H2O.18O/h;;;;;;;6*1H3;10*1H2;;;;;;;;;;;;;;;;;;/q;;;;3*+2;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;/p-6',
```

Comment:
InChI=1S/7Mo.6H3N.18H2O.10O/h;;;;;;;6*1H3;18*1H2;;;;;;;;;;/q;;;2*+1;2*+2;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;2*-1/p-4

DVXZHPYMEZBKFA-UHFFFAOYSA-J

Molecular Weight = 1254.1

Reply:
https://www.sigmaaldrich.com/IN/en/substance/ammoniumheptamolybdatetetrahydrate12358612054852?context=product

Reply:
Exisiting entry is good enough. No change needed